npm-packages/meteor-rspack/lib/localDependenciesHelpers.js

```
@@ -0,0 +1,180 @@
+ * @param {Function} callback - Function to call for each node
+ */
+function visitNode(node, callback) {
+  if (!node || typeof node !== 'object') {
+    return;
+  }
+
+  callback(node);
+
+  // Visit all properties of the node
+  for (const key in node) {
+    if (node.hasOwnProperty(key)) {
```

ESLint

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

(no-prototype-builtins)